index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon - SEO Experts</title>
+  </head>
+
+  <body>
+    <!--section containing header and nav bar embedded within containing links-->
+    <header class="header">
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <!-- section containing nav with individual links embedded leading to different parts of the site-->
+      <nav>
+        <ul>
+          <li>
+            <a href="#search-engine-optimizatoin">Search Engine Optimization</a>
+          </li>
+          <li>
+            <a href="#online-reputation-management"
+              >Online Reputation Management</a
+            >
+          </li>
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="hero"></div>
+    <div class="content">
```

Comment:
Ideally, this `div` container should of been a `main` tag, as this holds the main body of the webpage. `main` is also more semantic than `div`